htdocs/admin/pdf_other.php

```diff
@@ -69,6 +69,10 @@
 		dolibarr_set_const($db, "INVOICE_ADD_SWISS_QR_CODE", GETPOST("INVOICE_ADD_SWISS_QR_CODE", 'int'), 'chaine', 0, '', $conf->entity);
 		dolibarr_del_const($db, "INVOICE_ADD_ZATCA_QR_CODE", $conf->entity);
 	}
+	if (GETPOSTISSET('DOC_SHOW_SHIPPING_ADDRESS')) {
```

Comment:
Do you mean DOC_SHOW_SHIPPING_ADDRESS instead of DOC_SHOW_SHIPPING_ADDRESS ?

Reply:
I have a change @eldy